medical_physician/models/medical_physician.py

```diff
@@ -38,6 +38,17 @@ class MedicalPhysician(models.Model):
         comodel_name='medical.physician.schedule.template',
         inverse_name='physician_id',
     )
+    medical_center_primary_id = fields.Many2one(
```

Comment:
@lasley is the compute function only to add in the parent_id to medical_center_ids? Or is there something else that needs to be added to medical_center_ids as?

Should I also add an inverse function to the field as well?